examples/angular/src/app/pages/pages.component.html

```diff
@@ -1,18 +1,11 @@
-@if (context()?.pageAsset; as pageAsset) {
 <div class="flex flex-col min-h-screen gap-6 bg-neutral-50">
-
-    @if(pageAsset.layout.header) {
-        <app-header>
-            <app-navigation [items]="context()?.nav.children" />
-        </app-header>
-    }
-    
+    <app-header>
+        <app-navigation [items]="context()?.nav.children" />
+    </app-header>
+        
     <main class="container flex flex-col gap-8 m-auto">
-        <dotcms-layout [pageAsset]="pageAsset" [components]="components()" />
+        <dotcms-layout [pageAsset]="context()?.pageAsset" [components]="components()" />
```

Comment:
we have `pageAsset` and `pageInfo` and I'm proposing `pageData` we need to make sure this is unified.